### PR TITLE
Add Jumpbox support provided by BBL

### PR DESCRIPTION
**Pull request by henderjm**
Newer versions of BoshBootLoader create directors private to the world
and the only way to connect to them is through a provided Jumpbox

* Rename package names to reflect repository. This is very confusing
having references to old repository name.
* Add Jumpbox url, ssh key and username to source configuration
* Update dependency calls where function definitions have changed
* Update README

### CODE REVIEW

**Comment by ramonskie** [comment]
unfortunately i don't have bbl so i can't test it at the moment but it looks good.

**Comment by henderjm** [comment]
Hey, any update on this? It would be awesome to reduce our pipeline config a bit to use a supported resource :)

File: cmd/in/main.go [hunk 1]
```diff
@@ -4,11 +4,13 @@ import (
...
+	"log"
 	"os"
 
-	"github.com/starkandwayne/bosh2-errand-resource/bosh"
-	"github.com/starkandwayne/bosh2-errand-resource/concourse"
-	"github.com/starkandwayne/bosh2-errand-resource/in"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/bosh"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/concourse"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/in"
+	proxy "github.com/cloudfoundry/socks5-proxy"
```
Comment (7hunderbird) [inline cmd/in/main.go:13]: I'm still learning how go imports things.  Can you explain to me the syntax here for why we need the word `proxy` before the address to the `"github.com/cloudfoundry/socks5-proxy"` resource?

Comment (drnic) [reply cmd/in/main.go:13]: This allows you to alias the import. By default you’d reference this library by `socks5` I think.

Comment (7hunderbird) [reply cmd/in/main.go:13]: 👍, so we'd probably need the alias in each of the files that use the proxy.

* cmd/check/main.go
* cmd/in/main.go
* cmd/out/main.go

**Comment by rkoster** [comment]
@henderjm thanks!

**Comment by rkoster** [comment]
@henderjm changes released in: https://github.com/cloudfoundry-community/bosh2-errand-resource/releases/tag/v0.1.2

**Pull request merged**